compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -605,6 +605,10 @@ void addAMDAIEObjectFifoLoweringPasses(
   passManager.addPass(createCSEPass());
   passManager.addPass(createCanonicalizerPass());
 
+  passManager.addPass(createAMDAIEAssignTilesPass());
```

Comment:
Is this pass supposed to be added to the pipeline here? Thought you wanted to still do everything in `SplitLogicalObjFifosForConnectionReusePass`

Reply:
This pass is now run after `SplitLogicalObjFifosForConnectionReusePass`, so everything done in that pass (and the one from @yzhang93 we will enable soon) because those passes might cause us to want to assign new tile locations.

Reply:
Makes sense